build-system/tasks/validate-html-fixtures.js

```diff
@@ -72,13 +72,35 @@ async function getFileGroups(filesToCheck) {
   return fileGroups;
 }
 
+/**
+ * Checks for the existence of a local wasm / js validator binary and returns
+ * its location. Defaults to the wasm binary on the CDN.
+ * @return {Promise<string>}
+ */
+async function getValidatorJs() {
+  const localWasmValidatorPath =
+    'validator/bazel-bin/cpp/engine/wasm/validator_js_bin.js';
```

Comment:
@antiphoton Is this the correct path? If not, do you know what it is?

Reply:
Yes, this is the correct path.